internal/commands/app/create.go

```diff
@@ -21,7 +21,7 @@ import (
 var CommandMetaCreate = cli.CommandMeta{
 	Use:         "create",
 	Display:     "app create",
-	Description: "Create a new app from your current working directory and deploy it to the Realm server",
+	Description: "Create a new app or a template app from your current working directory and deploy it to the Realm server",
```

Comment:
[question] did this copy change come down as a requirement from design?  I know @kpham1231 was originally involved with crafting these messages and perhaps ighodalo has thoughts on what to put here

my gut?  probably would have expected `"Create a new app (or template app) from..."`

Reply:
cc: @Ighodaloijagbone@yahoo.com

Reply:
discussed offline: adding brackets is approved by design